content/dogepedia/how-tos/operating-a-node/index.md

```diff
@@ -0,0 +1,495 @@
+    shibetoshi:~$ dogecoin-qt
+    ```
+- Dogecoin Daemon is now starting.
+- You can now use the dogecoin-cli tool to interact with Dogecoin Core's JSON-RPC interface. Use the help command for a list of all available commands.
+    ```console
+    shibetoshi:~$ dogecoin-cli help
+    ```
+- To get help on a specific command:
+    ```console
+    shibetoshi:~ dogecoin-cli help getblock 
+    ```
+- You can follow the synching process by reading the contents of the debug.log file contained in the data directory. Go into the [data directory](/dogepedia/how-tos/operating-a-node/#data-directory), and type the following command:
+    ```console
+    shibetoshi:~$ tail -f debug.log
+    ```
+- In most setups, you can stop "listening" to the output of the debug.log file by pressing CTRL+C
+
+For more information on using Dogecoin CLI refer to the [**Dogecoin CLI**](/dogepedia/how-tos/operating-a-node/#dogecoin-cli) section. Additional configuration settings can be found in the [**Advanced Configuration**](/dogepedia/how-tos/operating-a-node/#advanced-configuration) section. For example, if you are not using Dogecoin Core as a wallet, it might be a good idea to disable the wallet functionality altogether.
+
+<hr />
+
+### Network Configuration {#network-configuration}
+By default, in most setups, your Dogecoin Core node might be able to establish only outbound connections. To support the Dogecoin network, you must allow inbound connections through port 22556.
+
+Depending on your local network setup, this might require extra configuration in your router and/or in your modem, and some extra rules added to your firewall.
+
+First of all, you will need to check whether incoming connections are working or not. When incoming connections are not enabled, Dogecoin Core will be able to establish only 8 outgoing connections. If the total connection count of your node is above 8, then it means your node is allowing incoming connections.
+
+You can check the number and type of connections very easily using Dogecoin Core GUI. If you look at the bottom right corner of the Dogecoin Core window, you will see an icon representing the current network status. Hover over the icon (do not click) and a popup should appear, displaying the total number of connections. 
+
+{{< figure src="windows-connection-hover-debug.png" alt="Dogecoin Core GUI - Hover To See Connection Count" >}}
+
+Otherwise, go into the Help -> Debug menu, and you should see a detailed overview of your network status.
+
+{{< figure src="windows-connection-debug.png" alt="Dogecoin Core GUI - Network Debug" >}}
+
+In both these examples, only incoming connections are enabled.
+
+If you are using Dogecoin CLI, you can check the total number of connections by using the *getconnectiontotal* command:
+
+```console
+shibetoshi:~$ dogecoin-cli getconnectioncount
+12
+```
+
+In this case, since there are more than 8 active connection, we know that the node is already allowing incoming connections. 

```

Comment:
change "connection" to "connections" (i.e., pluralise). :)

Reply:
Done! Thanks @RichieJohn!